examples/ex-normal.cpp

```diff
@@ -0,0 +1,436 @@
+//                       MFEM Example normal-bc - Parallel Version
```

Comment:
Could there be a serial version too?

Reply:
`ex-sliding` now has serial and parallel version `ex28` and `ex28p`. Note the serial solver in this case is pretty bad (no hypre).